tasks/i18n-download.js

```diff
@@ -0,0 +1,32 @@
+const fs = require('fs');
+const request = require('request');
+
+const download = function(uri, filename, callback){
+		request(uri).pipe(fs.createWriteStream(filename)).on('close', callback);
+};
+
+const base_url            = 'https://translate.wordpress.org/projects/wp-plugins/woocommerce-services/stable/';
+const filename_prefix     = __dirname + '/../i18n/languages/woocommerce-services-';
+const supported_languages = {
+	'ar': 'ar',
+	'es-mx': 'es_MX',
+	'es-ve': 'es_VE',
+	'es': 'es_ES',
+	'fr-ca': 'fr_CA',
+	'ja': 'ja',
+	'nl': 'nl_NL',
+	'ru': 'ru',
+	'pt-br': 'pt_BR',
+	'ro': 'ro_RO',
+	'sv': 'sv_SE',
+	'zh-cn': 'zh_CN',
+};
+
+for ( const locale in supported_languages ) {
+	const fileLocale = supported_languages[ locale ];
+	for ( const format of [ 'po', 'mo' ] ) {
```

Comment:
I tried to add `json` here so that `"prerelease": "npm run i18n && node tasks/i18n-download && npm run i18njson",` can be simply just `"prerelease": "npm run i18n && node tasks/i18n-download",`. Removing the `i18njson` script. 

I compared the 2 JSONs files and they look a little different. Not sure what the impact will be so I am going to create a separate issue to see if removing `i18njson` is possible.

Reply:
https://github.com/Automattic/woocommerce-services/issues/2066